src/Components/EmailPreview/index.jsx

```diff
@@ -0,0 +1,24 @@
+import './EmailPreview.css'
+
+const EmailPreview = ({name, subject, body, date_created, read}) => {
+    let dateObj = new Date(date_created)
+    let day = dateObj.getDate()
+    let month = dateObj.getMonth()
+    let year = dateObj.getFullYear()
+    let formattedDate = `${day}/${month}/${year}`
```

Comment:
why lets?

Reply:
also i think this needs to be in a useEffect to watch the date_created variable, as it needs to rerun if the prop changes